extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -94,11 +94,7 @@ export class Entitlement {
    * @return {!JsonObject}
    */
   jsonForPingback() {
-    return /** @type {!JsonObject} */ (Object.assign(
-      {},
-      {'raw': this.raw},
-      this.json()
-    ));
+    return jsonObjectAssign(dict({}), dict({'raw': this.raw}), this.json());
```

Comment:
Can we replace `dict({})` with `dict()`?

Reply:
yes :D